tests/test_all.py

```diff
@@ -111,7 +104,7 @@ def test_getrich_spotify_album():
     requested = requests.get(URL).text
     get_data = main_generic(requested, URL)
     enriched_data = enrich_test(URL)
-    assert enriched_data == test_enrich_spotify_album_output
+    # assert enriched_data == test_enrich_spotify_album_output

```

Comment:
@alihaiderrizvi asked me to comment this out. This test case has some issues. He'll fix it in the next PR.

Reply:
Cool no worries!